Excercise_3.js

```diff
@@ -1,2 +1,134 @@
 //Implement Singly Linked List.
 
+class Node {
+    constructor(val){
+        this.val = val;
+        this.next = null;
+    }
+}
+
+class SinglyLinkedList{
+    constructor(){
+        this.head = null;
+        this.tail = null;
+        this.length = 0;
```

Comment:
We need not have length and tail in constructor. Although they don't hurt yet all abstract functions can be implemented without these variables